invenio_drafts_resources/records/api.py

```diff
@@ -227,3 +228,15 @@ def edit(cls, record):
                 fork_version_id=record.revision_id,
             )
         return draft
+
+    @classmethod
+    def cleanup_drafts(cls, timedelta):
+        """Clean all the soft deleted drafts.
+
+        The drafts in the last timedelta spawn of time
+        won't be soft deleted.
+        """
+        seconds_ago = datetime.utcnow() - timedelta
```

Comment:
Comment:  Rename ``timedelta`` variable to something else, to avoid that it overwrites the timedelta module imported in the top. You can call it e.g. ``dt`` or ``delta``.

Also, the result of ``utcnow - delta`` is timestamp and not a delta so the naming ``seconds_ago`` doesn't fit well with the result. Something like ``timestamp`` fits better.